### PR TITLE
Jetpack VideoPress: Prepare release 1.1.0-beta

### PR DESCRIPTION
**Pull request by nunyvega**
Release for Jetpack VideoPress standalone plugin V1.1.0-beta

Testing instructions:

Read the changelogs and readmes.

### CODE REVIEW

File: projects/plugins/videopress/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.1.0 - 2022-11-22
```
Comment (sdixon194) [inline projects/plugins/videopress/CHANGELOG.md]:
```suggestion
## 1.1.0-beta - 2022-11-22
```

File: projects/plugins/videopress/readme.txt [hunk 2]
```diff
@@ -73,7 +73,27 @@ The file size limit is 5 GB. However, on slower networks, there is a chance the
 2. Edit your video details, cover image, and privacy from your VideoPress library.
 
 == Changelog ==
-### 1.0.0 - 2022-10-25
+### 1.1.0 - 2022-11-22
```
Comment (sdixon194) [inline projects/plugins/videopress/readme.txt]:
```suggestion
### 1.1.0-beta - 2022-11-22
```

**Comment by oskosk** [comment]
Thanks @sdixon194

**Review by oskosk — APPROVED** [review]

File: projects/packages/videopress/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,43 @@ All notable changes to this project will be documented in this file.
...
+- VideoPress: implement Privacy control [#27401]
+- VideoPress: improve when re-rendering the video player [#27546]
+- VideoPress: register and sync isPrivate block attribute [#27493]
+- VideoPress: Sync video tracks data in video block [#27495]
+- VideoPress: Updated the list of plans that have VideoPress included. [#27536]
+
+### Changed
+- Updated package dependencies. [#26069]
+- Updated package dependencies. [#26736]
+- Updated package dependencies. [#27043]
```
Comment (sdixon194) [inline projects/packages/videopress/CHANGELOG.md:28]: Probably want to remove the two duplicates.

File: projects/js-packages/idc/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 ### This is a list detailing changes for the Jetpack RNA IDC package releases.
 
+## 0.10.29 - 2022-11-22
+### Changed
+- Updated package dependencies. [#26069]
+- Updated package dependencies. [#26736]
+- Updated package dependencies. [#27043]
```
Comment (sdixon194) [inline projects/js-packages/idc/CHANGELOG.md:9]: Probably want to remove the two duplicates here.

**Pull request merged**